<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTextViewerWindow</class>
 <widget class="QMainWindow" name="CTextViewerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/main_icon</normaloff>:/main_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionAuto_detect_encoding"/>
    <addaction name="actionASCII_Windows_1252"/>
    <addaction name="actionSystemLocale"/>
    <addaction name="actionUTF_16"/>
    <addaction name="actionUTF_8"/>
    <addaction name="actionHTML"/>
    <addaction name="actionMarkdown"/>
    <addaction name="actionHex"/>
    <addaction name="separator"/>
    <addaction name="actionLine_wrap"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionFind_next"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSystemLocale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;System locale</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionUTF_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-&amp;8</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionUTF_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-1&amp;6</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>H&amp;TML</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_next">
   <property name="text">
    <string>Find next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionAuto_detect_encoding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto detect encoding</string>
   </property>
  </action>
  <action name="actionASCII_Windows_1252">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;ASCII (Windows-1252)</string>
   </property>
  </action>
  <action name="actionLine_wrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Line wrap</string>
   </property>
  </action>
  <action name="actionMarkdown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Markdown</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionHex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hex</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
